example/client/client.go

```diff
@@ -38,52 +40,56 @@ func main() {
 	dataset := flag.String("dataset", "opentelemetry", "Your Honeycomb dataset")
 	flag.Parse()
 
-	exporter, err := honeycomb.NewExporter(
+	exporter, errExp := honeycomb.NewExporter(
```

Comment:
Is there a reason to change this? If not, I'd prefer to leave it as it was, naming this `err` and using `err = func(ctx context.Context) error {}`  below.

Reply:
Ah, will fix that ... Just habit so err doesn't get shadowed.

Reply:
revert it back ... it is only example code :)